src/_data/whats-new.yml

```
@@ -103,6 +105,80 @@ entries:
+- description: Improved [REST API reference](https://devdocs.magento.com/guides/v2.3/rest/bk-rest.html)
+    performance.
+  versions: 2.3.x
+  type: Technical
+  date: June 22, 2020
+  link: https://github.com/magento/devdocs/pull/7430
+- description: Added the [Adding an admin grid](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/admin-grid.html)
+    topic.
+  versions: 2.3.x
+  type: New Topic
+  date: June 18, 2020
+  link: https://github.com/magento/devdocs/pull/7307
+- description: Added new configuration options to [Use Redis for Magento page and
+    default cache](https://devdocs.magento.com/guides/v2.4/config-guide/redis/redis-pg-cache.html).
+  versions: 2.4.0
+  type: Major Update
+  date: June 17, 2020
+  link: https://github.com/magento/devdocs/pull/7331
+- description: Removed Internet Explorer 11 from the list of [supported web browsers](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements_browsers.html).
+  versions: 2.4.0
+  type: Technical
+  date: June 17, 2020
+  link: https://github.com/magento/devdocs/pull/7419
+- description: Updated [Magento-developed extension (MDE)](https://devdocs.magento.com/release/#compatibility)
+    and [vendor-bundled extension (VBE)](https://devdocs.magento.com/extensions/vendor/#compatibility)
+    compatibility for Magento 2.4.0.
+  versions: 2.4.0
+  type: Major Update
+  date: June 17, 2020
+  link: https://github.com/magento/devdocs/pull/7423
+- description: Removed references to Signifyd, which was deprecated in 2.3 and removed
+    in 2.4.
+  versions: 2.4.0
+  type: Major Update
+  date: June 16, 2020
+  link: https://github.com/magento/devdocs/pull/7415
+- description: Added a banner message to the top of all pages in the [Payments Integration
+    Guide](https://devdocs.magento.com/guides/v2.4/payments-integrations/bk-payments-integrations.html)
+    (except [CardinalCommerce 3-D Secure](https://devdocs.magento.com/guides/v2.4/payments-integrations/cardinal/cardinal.html))
+    announcing the removal of the Braintree module in Magento 2.4.0 and updated links
+    to the Magento 2 codebase to point to the 2.3 branch.
+  versions: 2.4.0
+  type: Major Update
+  date: June 16, 2020
+  link: https://github.com/magento/devdocs/pull/7417
+- description: Removed references to deprecated payment methods Authorize.net, Cybersource,
+    eWay, and Worldpay. Replaced the Authorize.net payment example with a Paypal example
+    in the [Customize checkout](https://devdocs.magento.com/guides/v2.4/howdoi/checkout/checkout_payment.html)
+    tutorial.
+  versions: 2.4.0
+  type: Major Update
+  date: June 16, 2020
+  link: https://github.com/magento/devdocs/pull/7414
+- description: Reverted changes to the [2.3.x version](https://devdocs.magento.com/guides/v2.3/security/two-factor-authentication.html)
+    of the _Two Factor Authentication_ topic and updated the [2.4.x version](https://devdocs.magento.com/guides/v2.4/security/two-factor-authentication.html)
```

Does this have to change in the PR? If so, it's too minor to be concerned about. no worries.

```suggestion
    of the _Two-Factor Authentication_ topic and updated the [2.4.x version](https://devdocs.magento.com/guides/v2.4/security/two-factor-authentication.html)
```